### DISCUSSION TITLE
Unable to use exitERC721 function  { code: -32000, message: 'execution reverted' }

### DISCUSSION THREAD

**Discussion by Josedpv** [0xPolygon/matic.js#249] [Q&A]
You can see a detailed description of the error:
https://docs.google.com/document/d/1cBXjiGKdmMJ9bVLgCOPtxGthc1wEsApryUCP_a16IT0/edit

**Comment by ujjwalguptaofficial** [comment]
answered here - https://github.com/maticnetwork/matic.js/issues/250
